components/post_view/post_list/index.js

```diff
@@ -43,15 +44,18 @@ function makeMapStateToProps() {
         let atOldestPost = false;
         let formattedPostIds;
         let latestAriaLabelFunc;
-        const lastViewedAt = state.views.channel.lastChannelViewTime[ownProps.channelId];
-        const channelManuallyUnread = isManuallyUnread(state, ownProps.channelId);
+        const {focusedPostId, unreadChunkTimeStamp, channelId} = ownProps;
```

Comment:
0/5, but it might be better to avoid these sort of changes when you're not directly touching most of this code since it makes the diff messier. I'm not saying it needs to be changed back, but when not writing new code, I think it's a bit better to stick with the existing coding style (assuming it isn't absolutely awful, of course 😛)